lib/modules/manager/nix/index.ts

```diff
@@ -8,8 +8,8 @@ export const supportsLockFileMaintenance = true;
 export const url = 'https://nix.dev';
 
 export const defaultConfig = {
-  fileMatch: ['(^|/)flake\\.nix$'],
-  commitMessageTopic: 'nixpkgs',
+  fileMatch: ['(^|/)flake\\.lock'],
```

Comment:
didn't we told to revert this?

Reply:
Oh, yeah. With the fixes from @mfenniak we no longer need that.